code submission

shubham-04 opened the pull request:
Please check and let me know @shubhamsharma95@live.com

shubham-04 commented:
For Input please make changes in input.txt file. I have given sample input file.
To run the project copy whole thing in a project and run "node realimage.js" on cmd

maknahar commented:
Hi @shubham-04, Thanks for submitting the solution. Could you please remove all unnecessary debug logs from the program so the output is clean? Also, can we take the check command as command line argument?

shubham-04 commented:
Hi @maknahar  I have removed all debug logs and changed output to be saved in output.txt file.
I have code thinking that i will be getting all user input from input.txt so now i can't change it to command line input.
